example/build.yaml

```diff
@@ -0,0 +1,12 @@
+targets:
```

Comment:
Ditto – add a `# FYI ready about build.yaml here...` line at the top...

Reply:
Done